apps/badgeuser/models.py

```diff
@@ -326,8 +342,11 @@ def institution(self, value):
         :param value: Institution
         :return: None
         """
-        self.institution_set.add(value)
-
+        try:
+            InstitutionStaff.objects.get(user=self, institution=value)
+            raise ValueError('User already has an institution staff membership. Cannot have two.')
```

Comment:
Locally it returns 0 institution staff memberships, I think because you removed `is_staff` from the teacher seed and didn't replace it.